.github/workflows/release.yml

```
@@ -37,26 +36,23 @@ jobs:
       - name: Checkout Sources
-        uses: actions/checkout@v4
+        uses: actions/checkout@3d3c42e5aac5ba805825da76410c181273ba90b1 # v7
       - name: Download Artifacts
-        uses: actions/download-artifact@v4
+        uses: actions/download-artifact@3e5f45b2cfb9172054b4087a40e8e0b5a5461e7c # v8
         with:
           name: build-artifacts
           path: build-artifacts
       - name: Format Release Name
         id: format_release
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2129:style:3:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)